h/services/annotation_moderation.py

```diff
@@ -23,6 +23,24 @@ def hidden(self, annotation_id):
                         .filter_by(annotation_id=annotation_id)
         return self.session.query(q.exists()).scalar()
 
+    def all_hidden(self, annotation_ids):
+        """
+        Check which of the given annotation ids is hidden.
+
+        :param annotation_ids: The ids of the annotations to check.
+        :type annotation: list of unicode
+
+        :returns The subset of the annotation ids that are hidden.
+        :rtype set of unicode
```

Comment:
Ah right, my brain probably just doesn't want to add the trailing colons because it looks weird 😆